src/alarm_trap_sender.cpp

```diff
@@ -143,3 +166,112 @@ void AlarmTrapSender::send_trap(const AlarmTableDef& alarm_table_def)
 
   snmp_reset_var_buffers(&var_trap);
 }
+
+void AlarmTrapSender::send_enterprise_trap(const AlarmTableDef& alarm_table_def)
+{
+  static const unsigned int severity_to_state[] = {2, 1, 2, 6, 5, 4, 3};
+  static const oid snmp_trap_oid[] = {1,3,6,1,6,3,1,1,4,1,0};
+  static const oid trap_type_oid[] = {1,2,826,0,1,1578918,19444,9,2,1,1};
```

Comment:
Frig, that's my bad, I got the spec wrong. @AME2, you'll need to replace 1,2,826,0,1,1578918 with 1.3.6.1.4.1.9 (and I'll fix up the spec)